src/FastRouteRouter.php

```diff
@@ -80,15 +109,31 @@ class FastRouteRouter implements RouterInterface
      *     implementation will be used.
      * @param null|callable $dispatcherFactory Callable that will return a
      *     FastRoute dispatcher.
+     * @param array $config Array of custom configuration options.
      */
-    public function __construct(RouteCollector $router = null, callable $dispatcherFactory = null)
-    {
+    public function __construct(
+        RouteCollector $router = null,
+        callable $dispatcherFactory = null,
+        array $config = null
+    ) {
         if (null === $router) {
             $router = $this->createRouter();
         }
 
         $this->router = $router;
         $this->dispatcherCallback = $dispatcherFactory;
+
+        if (isset($config['cache_enabled'])) {
+            $this->cacheEnabled = (bool) $config['cache_enabled'];
+        }
+
+        if (isset($config['cache_file'])) {
```

Comment:
@Ocramius 
Hello Marco. Why class constants?
I meant those keys to be used (and related values loaded from) in configuration files, same way it's done for instance in

https://github.com/zendframework/zend-expressive-zendviewrenderer/blob/master/src/HelperPluginManagerFactory.php.

or like `config['middleware_pipeline']`, `config['routes']`, `config['db']`,, `config['db']['username']`, etc etc

If i write:
```php
if (isset($config[self::CONFIG_CACHE_FILE])) {
```
then I'd have to use it in config files too instead of a simpler `'cache_file'` config key. I've never seen class constants used in config keys in zend-expressive so far except for FQCNs in services/factories of course .

Or I am missing completely your point? :-)

kind regards